WhiteDentalClinic/WhiteDentalClinic.Application/Models/CustomerModel/UpdateCustomerResponseModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace WhiteDentalClinic.Application.Models.Customer
+{
+    public class UpdateCustomerResponseModel
+    {
+        public Guid Id { get; set; }
```

Comment:
On update usually the entire entity is returned on the response so that the caller can see the updates are indeed saved properly.